### PR TITLE
[CN-Translation]k8smeetup-tasks-administer-cluster-cpu-containt-namespace-4

### PR DESCRIPTION
**Pull request by xiaosuiba**
preview at:
https://k8smeetup.github.io/docs/tasks/administer-cluster/kubeadm-upgrade-1-8/
https://k8smeetup.github.io/docs/tasks/administer-cluster/cpu-constraint-namespace/
https://k8smeetup.github.io/docs/tasks/administer-cluster/developing-cloud-controller-manager/
https://k8smeetup.github.io/docs/tasks/administer-cluster//opaque-integer-resource-node/

/cc @zhangxiaoyu-zidif

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6304)

### CODE REVIEW

**Comment by steveperry-53** [comment]
@markthink

**Comment by chenopis** [comment]
@hanjiayao Is there anyone that can review this PR?

**Comment by chenopis** [comment]
@tengqm Would you be able to review these?

File: cn/docs/tasks/administer-cluster/opaque-integer-resource-node.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
+---
+cn-approvers:
+- xiaosuiba
+title: 为节点发布不透明整数资源（Opaque Integer Resources）
```
Comment (tengqm) [inline cn/docs/tasks/administer-cluster/opaque-integer-resource-node.md]: the OIR resource is deprecated... not need to translate this now.

**Comment by zacharysarah** [comment]
> not need to translate this now.

Closing!

**Pull request closed without merging**